packages/ihe-gateway-sdk/src/common/shared.ts

```diff
@@ -0,0 +1,20 @@
+import BadRequestError from "@metriport/core/util/error/bad-request";
+
+export function normalizeOid(oid: string | undefined): string {
+  if (!oid) throw new BadRequestError("OID must be present");
+  const oidRegex = /^[0-9]+(\.[0-9]+)*$/;
+  const urnOidRegex = /^urn:oid:([0-9]+(\.[0-9]+)*)$/;
```

Comment:
combine them both into a single regex pattern:

```^(urn:oid:)?([0-9]+(\.[0-9]+)*)$```

Reply:
oh this is cool! thx!